.codeclimate.yml

```diff
@@ -1,7 +1,12 @@
 ---
 exclude_paths:
 - ".git/"
+- "**.html"
+- "**.json"
+- "**.xml"
+- "**.yaml"
 - "**.yml"
+- "**.gif"
```

Comment:
I put images last, it looked odd with svg and gif mixed with json, yaml, yml, etc.